features/bevragen/persoon-beperkt/verblijfplaats/locatie/fields.feature

```diff
@@ -11,24 +11,24 @@ Functionaliteit: Persoon beperkt: verblijfplaats locatie
     | naam                  | waarde |
     | functie adres (10.10) | W      |
     En de 'verblijfplaats' heeft de volgende 'adres' gegevens
-    | naam                        | waarde                     |
-    | gemeente_code               | 0518                       |
-    | locatiebeschrijving (12.10) | Woonboot op de Grote Sloot |
+    | naam                        | waarde                      |
+    | gemeentecode (92.10)        | 0518                        |
+    | locatiebeschrijving (12.10) | Woonboot bij de Grote Sloot |
     Als personen wordt gezocht met de volgende parameters
     | naam          | waarde                              |
     | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
     | geslachtsnaam | Maassen                             |
     | geboortedatum | 1983-05-26                          |
-    | fields        | <field>                             |
+    | fields        | <fields>                            |
     Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
     | naam                      | waarde    |
     | type                      | Locatie   |
     | functieAdres.code         | W         |
     | functieAdres.omschrijving | woonadres |
 
     Voorbeelden:
-    | field                                    |
-    | functieAdres                             |
+    | fields                                   |
+    | verblijfplaats                           |
```

Comment:
In geval van dit eerste voorbeeld ook 'verblijfadres.locatiebeschrijving' teruggeven.

Reply:
Ook hier hetzelfde. LocatieBeperkt heeft alleen functieAdres veld (naast type)